### ISSUE TITLE
unreproducable

### ISSUE THREAD

**Issue by battlezzq** [callsys/TextVR#1]
Hello, i am interested in your excellent work(TextVR). But I can't run the code successfully
### Failed in Inference TextVR with given weights ckpt/textvr.pth
the result is below:
[t2v_metrics]TextVR epoch 0, R@1: 0.0, R@5: 0.1, R@10 0.2, R@50 1.1MedR: 1390, MeanR: 1369.0
[v2t_metrics]TextVR epoch 0, R@1: 0.0, R@5: 0.0, R@10 0.3, R@50 1.7MedR: 1376.5, MeanR: 1366.4

### Failed in Training TextVR with given config file configs/TextVR_fusion.json
I try to train the model, but failed again. 
[t2v_metrics]TextVR epoch 0, R@1: 0.0, R@5: 0.1, R@10 0.2, R@50 1.1MedR: 1390, MeanR: 1369.0
[v2t_metrics]TextVR epoch 0, R@1: 0.0, R@5: 0.0, R@10 0.3, R@50 1.7MedR: 1376.5, MeanR: 1366.4
The loss_0 is decrease to 0.75, and the val_loss_0 is 10.2, i wonder if my config is wrong or other reasons(over fitting?).

### Problem about the given example of sim_matrix.npy
i use the given sim_matrix.npy to get the result, but the result is below:
[t2v_metrics]TextVR epoch 0, R@1: 0.0, R@5: 0.1, R@10 0.2, R@50 1.1MedR: 1390, MeanR: 1369.0
[v2t_metrics]TextVR epoch 0, R@1: 0.0, R@5: 0.0, R@10 0.3, R@50 1.7MedR: 1376.5, MeanR: 1366.4
the code is below:
``` 
sims = np.load('/path/sim_matrix.npy')# 本地读取,using the given example of sim_matrix.npy
res = metric(sims) # , query_masks=mask
verbose(epoch=0, metrics=res, name="", mode=metric_name)
```

Thanks for your patience,i wonder if my config is wrong or other reasons.

**Comment by callsys** [comment]
The caption of the test set has been scrambled so that the model **cannot be validated offline**. If you want to verify a model's performance on the test set, submit the similarity matrix sim_matrix.npy to the [competition website](https://codalab.lisn.upsaclay.fr/competitions/13317).

**Comment by battlezzq** [comment]
Thanks for your reply.
I wonder if you would like to provide the true caption of the test set?
In my view, i cannot submit the similarity matrix sim_matrix.npy to the [competition website](https://codalab.lisn.upsaclay.fr/competitions/13317) due to its Competition Ends.

**Comment by callsys** [comment]
The [unscrambled annotation file](https://github.com/callsys/TextVR/blob/main/data/TextVR/TextVR_test.json) of the test set is released, replace `TextVR_test_rand.json` with `TextVR_test.json` for offline testing.

**Comment by battlezzq** [comment]
Thanks again！

**Comment by battlezzq** [comment]
```
{
        "path": "Street_View_Indoor/zxg9hOCpljUlistPLEU8J8CVRu8_V_RqmaaV_XQeCIXIC5lzindex15_02000_02010.mp4",
        "captions_info": [
            {
                "caption": "SLOANE PLACE is a boutique hotel and caf\u00e9 bar offering guests and locals relaxed elegance in a contemporary, chic setting.",
                "caption_info": {
                    "duration": 10.0,
                    "desc_id": 2161
                }
            }
        ],
        "**videos_info**": {
            "source": "vitvr_batch2",
            "split": "test",
            "Kwai_OCR": "Sports/G7ll8RnSgvU_00360_00370.json"
        }
    }
```

In the provided TextVR_test.json,
why the **videos_info** doesn't match with the video(**zxg9hOCpljUlistPLEU8J8CVRu8_V_RqmaaV_XQeCIXIC5lzindex15_02000_02010**)

**Comment by callsys** [comment]
We re-uploaded the correct TextVR_test.json.
